torch/onnx/verification.py

```diff
@@ -182,6 +198,7 @@ def _compare_ort_pytorch_outputs(
             raise
 
 
+@_beartype.beartype
```

Comment:
Same comments as before. Added for everything so that future authors are not confused about when some functions don't have the decorators and reviewers don't have to look for them every time. When annotations are added they will start to function automatically